java/src/main/java/com/google/oak/transport/GrpcStreamingTransport.java

```diff
@@ -67,8 +65,8 @@ public GrpcStreamingTransport(
   @Override
   public Result<AttestationBundle, String> getEvidence() {
     RequestWrapper requestWrapper = RequestWrapper.newBuilder()
-        .setGetPublicKeyRequest(GetPublicKeyRequest.newBuilder())
-        .build();
+                                        .setGetPublicKeyRequest(GetPublicKeyRequest.newBuilder())
```

Comment:
I don't think this reformatting should be part of this PR. If the current java formatting is incorrect (or if our formatting script uses the wrong format) that should be solved in a different PR.